ddl/db_integration_test.go

```diff
@@ -55,6 +55,7 @@ var _ = Suite(&testIntegrationSuite4{&testIntegrationSuite{}})
 var _ = Suite(&testIntegrationSuite5{&testIntegrationSuite{}})
 var _ = Suite(&testIntegrationSuite6{&testIntegrationSuite{}})
 var _ = SerialSuites(&testIntegrationSuite7{&testIntegrationSuite{}})
+var _ = Suite(&testIntegrationSuite8{&testIntegrationSuite{}})
```

Comment:
Please use SerialSuites because the test uses the global variable `TableIndexCountLimit`.

Reply:
done